test/test_ops.py

```diff
@@ -1665,6 +1665,24 @@ def test_one_hot(self):
     helper_test_op([], lambda: torch.nn.functional.one_hot(torch.tensor(data), 8).type(torch.int32),
                        lambda: Tensor(data).one_hot(8), forward_only=True)
 
+  def test_negative_log_likelihood(self):
+    data, target = torch.randn(1,3), [2]
+    helper_test_op([], lambda: torch.nn.functional.nll_loss(torch.nn.functional.log_softmax(data, dim = 1), torch.tensor(target)).type(torch.float32),
```

Comment:
Can you use helper_test_op like the other examples?

Reply:
Done with this, weights can be either None or a 1D tensor and helper_test_op would throw an error cause of that? correct me if im wrong.